spec/login_helper.rb

```diff
@@ -1,11 +1,10 @@
 def create_test_user_1
-    User.create(email: "testuser@makers.com", first_name: "test", last_name: "user", password: "abc123")
+  User.create(email: "testuser@makers.com", first_name: "test", last_name: "user", password: "abc123")
```

Comment:
Should probably remove this altogether and use one of the seeded database values

Reply:
Fine by me tbf